site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -705,7 +713,9 @@ public function showStatus($gradeable_id) {
                    // Total peer components to grade = Number of peer components * Number of Students who submitted
                    'total_who_submitted' => $total_who_submitted,
                    'total_components' => count($gradeable->getPeerComponents()) * $total_who_submitted,
+                   'non_late_total_components' => count($gradeable->getPeerComponents()) * $total_who_submitted,
```

Comment:
it looks like this is a old PR and there were some updates meanwhile, for instance, line 717 has been added, and now there is 'graded_components' in between 'non_late_total_components' and 'non_late_graded_components'.  it would be better style if they are put together.

Reply:
And for below lines too.